src/components/settings-menu/CurrencySection.js

```diff
@@ -39,7 +40,10 @@ const CurrencySection = () => {
   const onSelectCurrency = useCallback(
     currency => {
       settingsChangeNativeCurrency(currency);
-      reloadTimelines('PriceWidget');
+      // reload widget timelines only if on ios version 14 or above
+      if (ios && parseInt(Platform.Version, 10) >= 14) {
```

Comment:
```suggestion
      if (ios && parseInt(Platform.Version) >= 14) {
```

this is not needed here AFAIK